Maxxor.PCL/Result/MxResult.cs

```diff
@@ -56,7 +58,18 @@ public static MxResult<T> Ok<T>(T value)
         }
 
         #endregion
-        
+
+        #region WithSuccessMessage
+
+        public MxResult WithSuccessMessage(string successMessage)
```

Comment:
Without this method we can't access the Value property of Result<T> because a vanilla Result is returned. The test `WHEN_Result_has_a_type_SHOULD_return_Result_of_T_with_Value` fails without this methof.